bots/tests-scan

```diff
@@ -45,7 +45,7 @@ DEFAULT_VERIFY = {
 REDHAT_VERIFY = {
     "verify/rhel-7-5": [ 'master', 'rhel-7.5' ],
     "verify/rhel-7-5-distropkg": [ 'master' ],
-    "verify/rhel-7-6": [ ],
+    "verify/rhel-7-6": [ 'rhel-7.6' ],
```

Comment:
Why aren't we testing rhel-7.6 on PRs to the master branch, like rhel-7.5?

Reply:
No particular reason other than "let's not run too many tests for a PR". But with 7.6 being an important upcoming target, I think we actually should do this now. I'll send a separate PR about it, and disable rhel-7-5 in its stead (the -distropkg variant is the much more important one now).

Reply:
Done in #9746